packages/honeycomb-opentelemetry-web/package.json

```diff
@@ -2,14 +2,19 @@
   "name": "@honeycombio/opentelemetry-web",
   "version": "0.11.0",
   "description": "Honeycomb OpenTelemetry Wrapper for Browser Applications",
-  "main": "./dist/cjs/index.js",
```

Comment:
eslint doesn't support `exports` this _may_ cause breaking changes and we should call this out in the release notes. Details in the PR desc.

Reply:
What happens if we leave this in? Anyone not using a bundler aware of `exports` can't see the `/experimental` entry point?

Reply:
Nothing, probably. We can add that back in. `exports` is well-supported (except by eslint's import plugin, apparently).

Reply:
[81481d0](https://github.com/honeycombio/honeycomb-opentelemetry-web/pull/434/commits/81481d0c25a67ec8465d9f704dda068ef344ab5b)